lib/crowbar/client/app/upgrade.rb

```diff
@@ -219,12 +219,21 @@ def backup(component)
           catch_errors(e)
         end
 
-        desc "repocheck ADDON",
-          "Check for existing repositories for an addon (ha|storage)"
+        desc "repocheck COMPONENT",
+          "Check for existing repositories for a component (crowbar|ha|storage)"
 
         long_desc <<-LONGDESC
-          `repocheck ADDON` will check for existing repositories
-          for a specifix addon product. ADDON can be 'storage' or 'ha'
+          `repocheck COMPONENT` will check for existing repositories
+          for a specific component. COMPONENT can be 'crowbar', 'storage' or 'ha'
+
+          crowbar: checks for the repositories required to upgrade the crowbar
+          server
+
+          storage: checks for the repositories required to upgrade the nodes that
+          have the SUSE Enterprise Storage Addon installed
+
+          ha: checks for the repositories required to upgrade the nodes that
+          have the SUSE Enterprise High Availability Addon installed
```

Comment:
I'm fine with it in another PR, but see comment I'll add after this one.

Btw, SUSE Enterprise High Availability doesn't exist. It's SLE HA, not SE HA.